content/documentation/validations/index.adoc

```diff
@@ -87,6 +87,9 @@ This validation warning could be a result of duplicate definition of the same su
 
 Istio silently ignores the duplicate subsets and merge these destination rules without letting the user know. Only the first seen rule (by Istio) per subset is used and information from multiple definitions is not merged. While the routing might work correctly, this is most likely a configuration error. It may lead to a undesired behavior if one of the offending rules is removed or modified and that is probably not the intention of the deployer of this service. Also, if the two offending destination rules have different policies for traffic routing the wrong one might be used.
 
+icon:exclamation-triangle[] Istio highly recommend to always use FQDN for hosts on DestinationRules. However it allows to use other formarts like short names ('details' or 'details.bookinfo').
+Kiali only supports FQDN and simple service name as host formats: 'details.bookinfo.svc.cluster.local' or 'details' instead. Validations using the 'details.bookinfo' format might not be accurate.
```

Comment:
Kiali only supports FQDN and simple service names as host formats: for example, 'details.bookinfo.svc.cluster.local' or 'details'. 

(Should 'simple service names' be plural?)

"Validations using the 'details.bookinfo' format might not be accurate."  This implies that "details" is preferred over "details.bookinfo", is that what you intended?

Reply:
My goal to say in that section is that: "Use FQDN or simple name, don't use service.ns format". So the implication you mention is correct. details preferred over details.bookinfo but details.bookinfo.svc.cluster.local is preferred over details.
Thanks for helping!